addons/common/fnc_headDir.sqf

```diff
@@ -76,7 +76,11 @@ if (_unit != call CBA_fnc_currentUnit) then {
 private _diff = -_azimuth;
 
 if !(_offset isEqualTo "") then {
+#ifndef LINUX_BUILD
     ADD(_diff,_unit getDir ([_offset] call CBA_fnc_getPos));
+#else
+    ADD(_diff,[_unit, [_offset] call CBA_fnc_getPos] call BIS_fnc_dirTo);

```

Comment:
`ADD(_diff,[ARR_2(_unit, [_offset] call CBA_fnc_getPos)] call BIS_fnc_dirTo);`
this is how it must look like ARR_2 must be use because of some issues with the preprocessor.